timereport/app.py

```diff
@@ -80,7 +78,7 @@ def index():
         attachment = submit_message_menu(user_name, reason, date_start, date_end, hours)
         logger.info(f"Attachment is: {attachment}")
         slack_client_response = slack_client_responder(
-            token=config['slack_token'],
+            token=os.getenv('slack_token'),
```

Comment:
this change is very important for us! We keep our secrets in a encrypted travis file inside the .chalice/config.json file so we no longer have to store tokens in a config file. We will use a os.environ

Reply:
👍